### PR TITLE
Improve & restructure documentation a bit

### PR DESCRIPTION
**Pull request by tomkerkhove**
Improve & restructure documentation a bit

### CODE REVIEW

File: docs/auth/azure-key-vault.md [hunk 1]
```diff
@@ -3,11 +3,11 @@ title: "Authentication for Azure Key Vault"
 layout: default
 ---
 
-## Authentication
+# Authentication
```
Comment (stijnmoreels) [inline docs/auth/azure-key-vault.md]: Yes, not sure what the 'guidelines' say about this. I think there are other pages to (maybe on ohter repo's) that also uses the second header (`## `) as initlal header.

File: docs/features/key-vault/extensions/iconfiguration-integration.md [hunk 2]
```diff
@@ -3,12 +3,19 @@ title: "Replace configuration tokens with ISecretProvider"
 layout: default
 ---
 
-The `Arcus.Security.Providers.AzureKeyVault` package provides a mechanism to use your own `ISecretProvider` implementation when building your configuration for your application.
+# Replace configuration tokens with ISecretProvider
```
Comment (stijnmoreels) [inline docs/features/key-vault/extensions/iconfiguration-integration.md]: :+1:

File: docs/auth/azure-key-vault.md (hunk 1 above)
Comment (tomkerkhove) [reply docs/auth/azure-key-vault.md]: That is correct but I've aligned Web API already - https://github.com/arcus-azure/arcus.webapi/pull/57/files#diff-9b0c7b7749e7052a0df13b318231d12cR6

Comment (tomkerkhove) [reply docs/auth/azure-key-vault.md]: I think this makes more sense unless you disagree?

Can revert it, no problem.

Comment (tomkerkhove) [reply docs/auth/azure-key-vault.md]: Will merge and send new PR if it has to be changed.

Comment (stijnmoreels) [reply docs/auth/azure-key-vault.md]: No, it's good like it is. Just a question I had for previous and future docs.

**Pull request merged**